testing/endpoint.go

```diff
@@ -581,8 +581,46 @@ func (endpoint *Endpoint) ChanUpgradeInit(timeout channeltypes.Timeout) error {
 	// NOTE: this update must be performed after SendMsgs()
 	endpoint.ChannelConfig.Version = endpoint.GetChannel().Version
 
-	endpoint.Chain.Coordinator.CommitBlock(endpoint.Chain)
-	return endpoint.Counterparty.UpdateClient()
+	return nil
```

Comment:
how come this was removed?

Reply:
ah right. I forgot to leave a comment due to the time yesterday. Follows same pattern with rest of the channel endpoint functions, is there any downside to this? Apart from manually needed to update client etc after you call it.

Reply:
I actually like it updating the counter party client here, manually keeping track of client updates is useful for certain more granular tests, but I feel the main use case for this function will just be doing the regular handshake.

Reply:
can't it be an either/or then? If we update client at end of `UpgradeInit` or at the start of `UpgradeTry`, granted that we'll just be doing regular handshake it wouldn't make a difference no?

Happy to revert either way and go with what `UpgradeInit` established.

Reply:
aha, nice catch, I guess this provides a bit more flexibility. I see the addition below calling `UpdateClient` in `endpoint.ChanUpgradeTry` 👍

Reply:
This is just a question of whether `UpgradeInit` updates the client before returning or if `UpgradeTry` updates the client before attempting its logic? 

Both seem fine to me. As @DimitrisJim points out, updating the client before performing the next step (at the beginning of TRY) is how the other endpoint functions are structured. I think the benefit of this is when folks implement custom `endpoint.UpgradeInit` functions (such as gov gated) which don't update the client. (RegisterInterchainAccount is an example of this in the regular handshake case) Given these situations, it might make sense to just update before you perform the next step rather than preemptively